### PR TITLE
Updated num attempts and status columns for event questions

### PR DESCRIPTION
**Pull request by rachellegelden**
In the list of all questions for an event, remove num attempts and change status to submitted/not submitted for EXAM questions

### CODE REVIEW

File: canvas/models.py [hunk 1]
```diff
@@ -227,6 +227,9 @@ def is_closed(self):
     def is_not_available_yet(self):
         return self.start_date > timezone.now()
 
+    def is_exam(self):
```
Comment (keyvankhademi) [inline canvas/models.py:230]: add @property decorator here so you can directly use it in templates

File: canvas/templates/canvas/problem_set_snippet.html [hunk 2]
```diff
@@ -1,3 +1,6 @@
+{% load canvas %}
+{% is_exam_event event as is_exam %}
```
Comment (keyvankhademi) [inline canvas/templates/canvas/problem_set_snippet.html:2]: you dont need this anymore.

File: canvas/templates/canvas/problem_set_snippet.html [hunk 3]
```diff
@@ -7,7 +10,9 @@
         <th scope="col">Format</th>
         <th scope="col">Tokens&nbsp;Earned</th>
         <th scope="col">Status</th>
-        <th scope="col">Num&nbsp;Attempts</th>
+        {% if not is_exam %}
```
Comment (keyvankhademi) [inline canvas/templates/canvas/problem_set_snippet.html:13]: here you can use `{% if event.is_exam %}`

File: canvas/templates/canvas/problem_set_snippet.html [hunk 4]
```diff
@@ -19,10 +24,21 @@
             <td>{{ uqj.question.category.name }}</td>
             <td>{{ uqj.question.type_name }}</td>
             <td>{{ uqj.formatted_current_tokens_received }}</td>
-            <td>{{ uqj.status }}</td>
-            <td>
-                {{ uqj.num_attempts }} / {{ uqj.question.max_submission_allowed }}
-            </td>
+             {% if is_exam %}
```
Comment (keyvankhademi) [inline canvas/templates/canvas/problem_set_snippet.html:27]: `{% if event.is_exam %`

File: canvas/templates/canvas/problem_set_snippet.html [hunk 5]
```diff
@@ -19,10 +24,21 @@
...
+                {% exam_question_status event uqj as status %}
+                <td>
+                    {{ status }}
+                </td>
+            {% else %}
+                <td>
+                    {{ uqj.status }}
+                </td>
+            {% endif %}
+            {% if not is_exam %}
```
Comment (keyvankhademi) [inline canvas/templates/canvas/problem_set_snippet.html:37]: `{% if not event.is_exam %`

File: canvas/templatetags/canvas.py [hunk 6]
```diff
@@ -40,3 +40,16 @@ def event_button_text(event, user):
         return "Not Available"
     else:
         return "Open"
+
+
+@register.simple_tag
+def is_exam_event(event):
```
Comment (keyvankhademi) [inline canvas/templatetags/canvas.py:46]: You don't need this anymore

**Comment by rachellegelden** [comment]
Closing for rebase

**Pull request closed without merging**